src/Renderer.js

```diff
@@ -1,21 +1,25 @@
-export const FORME_RECT = 0;
-export const FORME_CIRCLE = 1;
-export const FORME_TRIANGLE = 2;
+export const FORMS = {
+    RECT: 0,
+    CIRCLE: 1,
+    TRIANGLE: 2
+};
+
+const formsFunc = {
+    0: (c, r) => c.rect(r.x, r.y, r.width, r.height),
```

Comment:
Reverted to earlier code as it was kinda useless!
Every "() =>" will be translated to "function{...}" so I am not sure we would gain anything with it!